### ISSUE TITLE
Publish original Marian configs

### ISSUE THREAD

**Issue by La0** [mozilla/firefox-translations-training#313]
Depends #309 

The publication code only supports Marian final configuration (from its own output).
But it would be useful to view the hierarchy of YAML files directly from Weight & Biases.

The hiearchy of files is merged in a specific order, so we need to investigate:
- how to get a final unique YAML file
- can we isolate the parsing code & run it from publication code
- or simply upload the YAML files as they are on-disk

**Comment by La0** [comment]
@eu9ene @gregtatum If you could provide links to which config files are used at runtime so we can publish them :pray:

**Comment by eu9ene** [comment]
Here are the two configs that we pass to marian:
https://github.com/mozilla/firefox-translations-training/blob/5ccc2c396cc2992095b43851cb1c5d47bd4f0d5a/pipeline/train/train.sh#L138

The extra parameters that we can save as a separate config yaml: https://github.com/mozilla/firefox-translations-training/blob/5ccc2c396cc2992095b43851cb1c5d47bd4f0d5a/pipeline/train/train.sh#L157

And the OpusTrainer config: https://github.com/mozilla/firefox-translations-training/blob/5ccc2c396cc2992095b43851cb1c5d47bd4f0d5a/pipeline/train/train.sh#L133

So basically we should just upload as files whatever is passed to OpusTrainer/Marian.

**Comment by La0** [comment]
@vrigal mentionned that the whole line to start marian is actually logged, and should be parseable. It contains all the YML files.

**Comment by eu9ene** [comment]
As we discussed the paths to Marian configs are present in the args and for the OpusTrainer one we can just read it from the disk using the fixed path.
